### PR TITLE
Workshop page updates

### LINKED ISSUE #538: Clean up design of CSS Workshop page so that it's more skimmable
@mirisuzanne Ready for review/implementation
[Figma File Design Proposal](https://www.figma.com/file/iQN05m0jxjl60M6qpEjMW4/Cascading-Style-System-Workshop?type=design&node-id=1-11&mode=design&t=gVd49aCSy1KaAY1v-0)

### PR DESCRIPTION
**Pull request by mirisuzanne**
- [x] Initial pass at content and layout
- [x] Finalize 'what you get' with icons
- [x] Finalize pricing
- [x] Finalize contact-form / CTA design
- [ ] Finalize video intro

## Related Issue(s)

- Closes #538 

## Steps to test/reproduce

- /workshops/cascading-styles-deep/

### CODE REVIEW

File: content/workshops/cascading-styles-deep.md [hunk 1]
```diff
@@ -90,12 +126,19 @@ with at least a basic understanding in HTML and CSS.
 You don’t need to be an expert to keep up,
 but even the experts are likely to learn something new.
 
-## Schedule
+[Schedule a call »](#contact-us-to-book-a-workshop)
```
Comment (stacyk) [inline content/workshops/cascading-styles-deep.md:129]: I wonder if we want to edit this label a bit. The link takes you to the "Contact Us to Book a Workshop" form but I know next to the submit button there is a link to schedule a call. But also, "schedule a call" feels more like we are going to discuss a project... maybe something in the realm of of the link url "Contact us to book a workshop" or add in discuss the details of the workshop or something?

File: content/workshops/cascading-styles-deep.md [hunk 2]
```diff
@@ -151,6 +204,11 @@ and interactive exercises related to distributing space.
 
 ### 4. Intrinsic Layouts: Defining Structure
 
+{{ embed.figure([{
+  img: 'pages/css-workshop/grids.jpg',
+  alt: 'Cat in a box, thinking OMG I have so many questions for this damn box'
```
Comment (stacyk) [inline content/workshops/cascading-styles-deep.md:209]: lol

File: content/workshops/cascading-styles-deep.md [hunk 3]
```diff
@@ -167,6 +225,11 @@ and interactive exercises related to defining layouts.
 
 ### 5. Resilient Styles: Organizing Conventions
 
+{{ embed.figure([{
+  img: 'pages/css-workshop/conventions.jpg',
+  alt: 'A woman in a suit kneels to pet cute ducklings while the mother duck removes a bill from her purse'
```
Comment (stacyk) [inline content/workshops/cascading-styles-deep.md:230]: `money` instead of `bill`? Ducks have their own bills 🤣

File: src/scss/patterns/_icon-blocks.scss [hunk 4]
```diff
@@ -1,9 +1,21 @@
...
+[icon-grid] {
   --fill: var(--icon);
+}
+
+[icon-grid='full'] {
+  --media-size: calc(var(--extra) - var(--gutter));
   --column: full;
 }
 
+[icon-grid='fit'] {
```
Comment (stacyk) [inline src/scss/patterns/_icon-blocks.scss:10]: I wonder if we want to allow some icon blocks, like this one, to have left aligned (but still on top) icons as it looks a bit off balanced if we have smaller amounts of text at small (not tiny) screens 
Screenshot 2024-04-03 at 2 16 54 PM

**Review by stacyk — APPROVED** [review]
Left a few thoughts but it looks good. I wonder if we want to consider adding the featured workshop to the homepage? It isn't the easiest thing to find, but we could always link directly to it if we are sending the link out. 

Also, I wonder if we want another call to action at the bottom of the featured workshop content on the workshop single (above events)?

**Review by dvdherron — APPROVED** [review]
One little note about layout but otherwise this LGTM.

**Comment by SondraE** [comment]
@mirisuzanne @stacyk 
Another place we could add a link and CTA about this workshop is the Susy page: https://www.oddbird.net/susy/. Over the last 12 months, the Susy page is still the top most visited on OddSite. 

We already have language on that page that is related to training: "If you need help moving off Susy, or learning the latest in web layout, we offer [training](https://www.oddbird.net/talks/) and [consulting](https://www.oddbird.net/services/development/) to help bring you up-to-date." Even with the bold weight, that message is kinda buried in that first paragraph. If you don't read that paragraph thoroughly, the "contact us for details" CTA isn't super clear. The consulting link goes to our design and development services page which is kinda related to consulting, but might be a surprising place to land as a user. In conclusion, this Susy intro paragraph and CTA could use some re-thinking and potentially direct people to the CSS Workshop, but perhaps this should happen in a separate story.

**Review by SondraE — CHANGES_REQUESTED** [review]
- Would it be difficult to move the Featured Workshop section on the Talks list page (that features this CSS Workshop) above the quotes? I keep having this sense that we should let visitors know what the workshop content is they can hire us for, before we validate our workshop credentials with testimonials.
- What would you think of using the same colors for CTA links that we use for all other links -- blue text and pink underline. When I look at OddSite with a soft focus, those blue/pink links seem to stand out more than the links that are totally pink. 
- I like your idea of adding bold text to the bulleted list. You could also remove 1 or 2 points, especially since we go into more detail in the outline. Is there one of those points that could be pulled out and featured somehow? Like a pull quote in a magazine?
- I agree that there should be a CTA at the bottom of this page
- The "schedule a call" CTA right under the "who is this for" section is easy to miss. Could there be more space between that CTA and the outline section? Actually maybe no CTA is needed here, especially if there's another one at the bottom of the page. Also the CTA words should be "contact us to book a workshop" or "bring this workshop to your company" since the link goes to the contact form. You could keep "schedule a call" if you send people to the calendly site, but I think sending them to the in-page contact form is preferable.

**Comment by mirisuzanne** [comment]
I think I've addressed all the review here - setting aside the comments that might make more sense in a new PR. Happy for re-review.

**Review by SondraE — APPROVED** [review]
- It just struck me that "Pick a time to meet" could be misunderstood as picking a time for the workshop to occur, especially because it's right beside "Book Now." Maybe reword to "schedule intro call" or "schedule a call to book" or "schedule a call to learn more"
- Since I just recommended a bit of a different orientation for the intro video, it make me  think we may want to include a "Workshop Benefits" or "How This Workshop Benefits Companies" or "Benefits for Companies" section that lists the benefits in a way that will speak directly to CEOs, CTOs, COOs. Bullet points like: 1. Easier to scale, 2. Efficient and effective developer team, 3. A delightful and consistent experience for your customers, 4. More accessible website, 5. Updates to your web software that get the biggest bang for your buck
- Can we add one sentence to the intro paragraph that gets to the reason why a CEO might want to bring this workshop to their company? Something about not wanting to get stuck with old code that gets more expensive to update and more difficult to scale.
- Not to add even more, but we should probably create another card to add a speaker bio section about you.

File: content/_includes/birds.macros.njk [hunk 5]
```diff
@@ -101,3 +103,47 @@ params:
...
+      } -%}
+
+      {# This should be using the `h` shortcode #}
+      <h3 {{ utility.show_attrs(attrs) }}>
+        {{ utility.link_if(
+          content=bird.data.title | mdInline | safe,
+          url=bird.page.url if link else none,
+          class='title-link u-url'
+        ) -}}
+      </h3>
```
Comment (mirisuzanne) [inline content/_includes/birds.macros.njk:136]: @jgerigmeyer for some reason, I get an error when I try to use the `h` shortcode here. Thoughts?

**Review by dvdherron — APPROVED** [review]
changes look good to me 👍🏽

**Review by stacyk — APPROVED** [review]

File: content/workshops/cascading-styles-deep.md [hunk 6]
```diff
@@ -4,9 +4,10 @@ sub: A workshop on resilient & maintainable CSS
 feature: workshop
 author: miriam
 date: 2023-01-09
-action:
-  text: Hire us to give a workshop »
-  url: /contact/
+eleventyComputed:
+  action:
+    text: Bring this workshop to your company »
+    url: '{{ page.url }}#contact-us-to-book-a-workshop'
```
Comment (jgerigmeyer) [inline content/workshops/cascading-styles-deep.md:10]: Could this just be:

```suggestion
action:
  text: Bring this workshop to your company »
  url: '#contact-us-to-book-a-workshop'
```

File: content/_includes/birds.macros.njk [hunk 7]
```diff
@@ -101,3 +103,48 @@ params:
...
   {%- endif -%}
 {%- endmacro %}
+
+{% macro card(
+  bird,
+  birds=none,
+  level=3,
+  link=true,
+  bio=false,
+  class=''
```
Comment (jgerigmeyer) [inline content/_includes/birds.macros.njk:113]: It looks like we're not using any of these in practice -- do we want to remove some (e.g. `bio`), or do we think we'll use them?

File: content/_includes/birds.macros.njk (hunk 5 above)
Comment (jgerigmeyer) [reply content/_includes/birds.macros.njk]: There seems to be an issue using the shortcode inside the `call` block. I avoided it by adding a `set` in https://github.com/oddbird/oddleventy/pull/548/commits/a7c8a2a141f647385601f40621a982966365a3bf

File: content/workshops/cascading-styles-deep.md [hunk 8]
```diff
@@ -34,68 +35,116 @@ events:
...
     url: https://smashingconf.com/freiburg-2023/workshops/miriam-suzanne/
+# placeholder media
+intro:
+  media:
+    - iframe: https://player.vimeo.com/video/691514550
+      span: full
+  caption: |
+    Join Miriam for a deep dive
+    on developing **resilient and delightful experiences**
+    that hold up across browsers, languages, and device interfaces.
```
Comment (jgerigmeyer) [inline content/workshops/cascading-styles-deep.md:46]: Looks like this isn't used (yet)?

Comment (mirisuzanne) [reply content/workshops/cascading-styles-deep.md:45]: Yeah, we can remove for now. We need to record a new video, and don't want to wait for it to go live.

File: content/_includes/birds.macros.njk (hunk 7 above)
Comment (mirisuzanne) [reply content/_includes/birds.macros.njk]: Good catch. I built this out for using on the workshop page, and then it didn't look great, so I abandoned it. Bio is removed. I think the others still help make it re-usable if we ever want.

**Comment by mirisuzanne** [comment]
@jgerigmeyer I think addressed your review here

File: content/workshops/cascading-styles-deep.md [hunk 9]
```diff
@@ -34,68 +34,107 @@ events:
...
+## Workshop Outline
+
+The workshop is taught in five sessions:
 
 ### 1. Resilient Styles: a Declarative Cascade
 
+{{ embed.figure([{
+  img: 'pages/css-workshop/cascade.jpg',
+  alt: 'Yoda reaching out to control the force, surrounded by jungle swamp'
+}]) }}
```
Comment (jgerigmeyer) [inline content/workshops/cascading-styles-deep.md:136]: Are these placeholder images, or are we planning to publish them?

Comment (mirisuzanne) [reply content/workshops/cascading-styles-deep.md:136]: These are the images we used for the modules in the workshop site - they come from the slides. The plan was to publish them.

**Review by jgerigmeyer — APPROVED** [review]
I'm not clear if the earlier review comments from others have been addressed or not, but this LGTM.
